packages/amplify-e2e-core/src/index.ts

```diff
@@ -65,7 +65,7 @@ export async function createNewProjectDir(
   } while (fs.existsSync(projectDir));
 
   fs.ensureDirSync(projectDir);
-  console.log(projectDir);
+
```

Comment:
I'd perfer to leave this in as I use it for debugging sometimes

Reply:
noconsolelog eslint, when I need it I usually add it temporarily just like with other tests.